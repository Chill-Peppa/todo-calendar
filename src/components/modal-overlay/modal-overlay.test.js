//тут надо уже с функционалом писать блин
