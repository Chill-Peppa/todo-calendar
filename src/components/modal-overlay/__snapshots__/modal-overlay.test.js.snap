// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should correctly render App 1`] = `
<div
  className="modal__background"
  onClick={[Function]}
/>
`;

exports[`should correctly render ModalOverlay 1`] = `
<div
  className="modal__background"
  onClick={[Function]}
/>
`;
